tests/conftest.py

```diff
@@ -150,3 +159,17 @@ def fixture_bright_image():
     # Create a bright image (50x50 pixels, almost white)
     bright_img_array = np.full((50, 50, 3), 255, dtype=np.uint8)
     return Image.fromarray(bright_img_array)
+
+
+@pytest.fixture
+def fixture_nuclear_speckle_example_image():
+    # create an image array from example nuclear speckle data
+
```

Comment:
Is there supposed to be code directly under this code comment?

Reply:
Thanks, removing the extra line (the comment pertains to the line below).